.travis/config.sh

```diff
@@ -69,6 +69,10 @@ function ks_lsr_install_tuned() {
 
 # install tuned only where needed
 KS_LSR_PYVER=$(lsr_get_python_version $(type -p python))
+if lsr_compare_versions $KS_LSR_PYVER -lt 2.7 ; then
+  lsr_info kernel_settings tests not supported on python $KS_LSR_PYVER - skipping
```

Comment:
Could you please leave a comment why is it unsupported? Is it because the tests are unsupported on Python 2.6, or that the module code itself does not support Python 2.6 (I see that you exclude EL6 - the main reason why one wants to support Python 2.6 - here: https://github.com/linux-system-roles/kernel_settings/pull/1/files#diff-9b4ddf055111f4371f04fbb8b5366655R14 so it is probably the latter, but the wordings suggests the former.)

Reply:
The main problem is that not all of the dependencies for running tuned using python 2.6 are available via pypi.  For example, you cannot install `PyGObject` from pypi - none of the versions available on pypi will install on python 2.6.